app/views/comments/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<section>
+  <h1 class="users-title">Add comment</h1>  
+  <%= form_with model: @comment, url: user_post_comments_path, class: "card left" do |form| %>
+    <%= form.label :text %> <br>
+    <%= form.text_area :text , cols: 40, rows: 10 %> <br>
+    <%= form.submit "Submit", class: "btn" %>
+  <% end %>
```

Comment:
✅ There is a form in the views/comments/new.html.erb file which accepts input for creating comments 👍🏻